### ISSUE TITLE
[ BUG REPORT ] - Can't snap at snap.

### ISSUE THREAD

**Issue by VendixDilan** [Just-Natsuki-Team/NatsukiModDev#654]
# Overview:
I couldn't call Snap when i played with Nat, and neither did she. I won because her cards ran out.
It stayed until i restarted the game, after that it worked normally.

## Replication steps:
I asked her if we can play snap again. Before that she got mad at me for "cheating" and i apologized for it.
And like i said, for some reason I couldn't call snap.

## Additional information:
I'm not sure what's considered Important in this case, so i'll just list some details.
I played in Windowed mode.
 My OS is Windows 10.
 I've the newest version of JN (1.1.1).
![Bild_2023-04-28_184657529](https://user-images.githubusercontent.com/64294773/235206291-bded397b-14b0-4e10-96c7-fbaecac73825.png)

**Comment by Obsidian-Knife** [comment]
Hey there, thanks for bringing this to our attention.

Do you happen to remember how you tried to call snap? Was it with mouse input or with the more recently added hotkey input via your keyboard? Or perhaps try both when it happened and neither worked?

**Comment by VendixDilan** [comment]
I used the mouse to click on the button.
I didn't know about the keyboard input so i couldn't try that.

**Comment by RaionArt** [comment]
Hello, could you clarify a bit more? 
If Natsuki wasn't able to call snap and you neither, how did the game end? Both of you just kept laying cards until no cards were left for Natsuki?

**Comment by VendixDilan** [comment]
Exactly, it ended in me winning because Natsuki had no cards left.
And the bug still was there in the game after that one, until i restarted the game, then it was fixed.

**Comment by RaionArt** [comment]
Winning after the opponent has no cards left is intentional and the basic rule of the snap game. But what exactly was the bug? 
The false accusing of cheating is something we should take a look at, I agree with that.

**Comment by VendixDilan** [comment]
Like i said, i couldn't call Snap when playing and neither did she.
The only thing i could do was forfeit or place another card.
As shown in the picture, it was my turn and i couldn't choose "Snap", it was grayed out.

**Comment by VendixDilan** [comment]
So thanks to another user on Discord, i found it's caused by her calling you a cheater.
To clearify what happend in the video. I snapped too many times, i did that over and over until she got mad at me, she told me to apologize before asking her to play again. So i did, i apologized, after that i couldn't call snap again, not with the mouse or with the Hotkey. I tried forfeiting and starting a new round of snap. Both didn't fix it, it only went away after i restarted the game.
Is there any way i can include a video that is larger than 10MB? I've recorded an example.
